lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
+  ReactDOM.render(
```

Comment:
Definitely not necessary for this pull request, but let's think about having a central place to handle initializing an rendering components like we do [in the voting app](https://github.com/DoSomething/voting-app/blob/04234aebff2cf60114e3d0c36dd02d25d2fe555b/resources/assets/js/app.js#L54-L60).